bio/mapdamage2/environment.yaml

```
@@ -0,0 +1,6 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - mapdamage2 =2.2.1=pyr40_0
```

Please just pin to the version, not the build string (build strings are not guaranteed to stay valid forever).
```suggestion
  - mapdamage2 =2.2.1
```